ktor-features/ktor-gson/jvm/src/io/ktor/gson/GsonSupport.kt

```diff
@@ -47,7 +48,7 @@ public class GsonConverter(private val gson: Gson = Gson()) : ContentConverter {
 /**
  * Register GSON to [ContentNegotiation] feature
  */
-public fun ContentNegotiation.Configuration.gson(
+public fun ContentNegotiationConfig.gson(
```

Comment:
Could you tell me why we need this change?

Reply:
Because `ContentNegotiation` now is just a variable that has a type of `KtorPlugin` and `ContentNegotiationConfig` is a configuration for it.